geth/jail/jail.go

```diff
@@ -90,17 +90,17 @@ func (j *Jail) createCell(chatID string) (*Cell, error) {
 	}
 
 	cell := NewCell(chatID)
-
-	err := j.initCell(cell)
-	if err != nil {
-		return nil, fmt.Errorf("cell init failed: %v", err)
-	}
-
 	j.cells[chatID] = cell
 
 	return cell, nil
 }
 
+// CreateCell creates a new cell. It returns an error
+// if a cell with a given ID already exists.
+func (j *Jail) CreateCell(chatID string) (common.JailCell, error) {
```

Comment:
What is this wrapper for? If we need `createCell` to be exported, we can just export it, no?

Reply:
In the interface, this method must return `common.JailCell` while `createCell` returns `*Cell`.

Those interfaces in `common/types.go` is another thing to refactor...

Reply:
Got it, thanks!